modules/text_on_image/text_on_image_100_Jasmine_Davidson.html

```diff
@@ -0,0 +1,43 @@
+<div class="image-container">
+    <img src="https://images01.nicepagecdn.com/page/55/71/html-template-5571820.jpg" alt="Background Image">
```

Comment:
You used the link of the image you were suppose to design.

Reply:
your background image is really not a background image as it contains other text and write up...